.github/stale.yml

```diff
@@ -0,0 +1,25 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 120
```

Comment:
I vote to set this to `daysUntilStale: 90`. As far as I can tell we'll still mark the same 4 issues as stale initially at either 90 or 120 days, so might as well tighten it up somewhat:

https://github.com/linkerd/linkerd2/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+updated%3A%3C2018-06-03

https://github.com/linkerd/linkerd2/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+updated%3A%3C2018-07-03

(excluding the 2 `area/docs` issues that appear in those results)